tests/src/strategy.spec.ts

```diff
@@ -12,7 +12,13 @@ describe('Strategy with passport', () => {
   const request = supertest(app);
 
   // This will trigger the auth flow
-  app.get('/auth', passport.authenticate('snyk-oauth2', { state: 'test' }));
+  app.get(
+    '/auth',
```

Comment:
I can add an extra test case in this file for the current static nonce vs the new dynamic one if this fix is OK.

Reply:
I believe this won't work since `passport.AuthenticateOptions` does not have `nonce` defined?

Reply:
It's a TypeScript thing. We may need to change some bits here and there in Passport (Snyk's sponsoring them btw) to make it work. But it doesn't affect the behaviour.